docs/user/index.md

```diff
@@ -0,0 +1,185 @@
+# User & Programmers Manual
+
+This document describes how to use Perseo CEP.
+
+## Content
```

Comment:
This file seems to be an evolved version of old plain_rules.md (I haven't compared then in deep but some sections seem to be the same, i.e. the one for metadata). Is this correct?

Reply:
We were trying to create a more comprehensive and friendly user / programmer guide for RTD. Some parts may match paragraphs in the plain_rules file

Reply:
I see it includes:

![imagen](https://user-images.githubusercontent.com/1534240/83750545-be219100-a665-11ea-944d-d8616e08f95f.png)

so the detail is still containing in plain_rules.md and this file is a kind of "facade" for it. Is my interpretation correct?